src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const rules = sourceString
+    .split(/;(?![^(]*\))/)
+    .filter((rule) => rule.trim() !== '');
+
+  rules.forEach((rule) => {
```

Comment:
The use of `forEach` is allowed according to the checklist, but consider using `reduce` to transform the array directly into an object, which can be more efficient and declarative.